en/asgardeo/docs/apis/organization-apis/restapis/organization-user-share.yaml

```
@@ -0,0 +1,638 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: 'User Sharing API Definition'
+  description: |-
+    This API provides organization administrators with the ability to manage user access across child organizations. It supports operations to share users with specific or all child organizations, retrieve shared access details, and remove shared access as needed. The API also includes features for paginated retrieval of organizations and roles associated with shared users.
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: iam-dev@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://{server-url}/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: "carbon.super"
+      server-url:
+        default: "localhost:9443"
+paths:
+  /users/share:
+    post:
+      tags:
+        - User Sharing
+      summary: Share a user across specific organizations
+      description: |
+        This API shares one or more users across specified organizations, assigning roles based on the provided policy. The policy defines the sharing scope for each organization, including whether access extends to child organizations.
+        
+        <b>Scope(Permission) required:</b> `internal_user_share`
```

```suggestion
        <b>Scope(Permission) required:</b> `internal_org_user_share`
```
update scopes of organization APIs

---

Addressed with commit: [43e1da0](https://github.com/wso2/docs-is/pull/5026/commits/43e1da0ce010696a8afe7e3a1b21a9b39cf9732f)